src/ontology/mondo-edit.obo

```diff
@@ -128134,23 +128134,18 @@ property_value: confidence "5.666666666666667" xsd:double
 
 [Term]
 id: MONDO:0007245
-name: neurofibromatosis type 6
-def: "Neurofibromatosis type 6 (NF6), also referred as cafe-au-lait spots syndrome, is a cutaneous disorder characterized by the presence of several cafe-au-lait (CAL) macules without any other manifestations of neurofibromatosis or any other systemic disorder." [Orphanet:2678]
+name: cafe au lait spots, multiple
+def: "A cutaneous disorder characterized by the presence of several cafe-au-lait (CAL) macules without any other manifestations of neurofibromatosis or any other systemic disorder." [Orphanet:2678]
```

Comment:
hmm, do we not have a standard way of saying this is adapted rather than verbatim from orphanet, eg add your orcid?

Reply:
No, we haven't been doing this, but we can do going forward.